### PR TITLE
Week 9

### PR DESCRIPTION
**Pull request by tehvicke**
Link to my deployed project:
https://objective-leavitt-7853c0.netlify.com/

My reflections on how this week's project turned out:
Good! Didn't manage to get the shading correct for the play circle (could possibly be solved by adding another div with some css magic)

Things I'd like to have clarified or explained in more detail:
Nothing in particular.

### CODE REVIEW

File: code/src/Components/Album.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    if (index !== ArtistList.length - 1) {
+      output.push(', ')
+    }
+    console.log(output)
+  })
+ 
+  return (
+    <div class="album">
+      <div class="image">
+        <img src={props.img} alt="Album"></img>
```
Comment (ericaglimsholt) [inline code/src/Components/Album.js]: For the `alt` here maybe you can use the title or something to describe what the image is?

File: code/src/Components/Album.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="overlay">
+          <div class="interact">
+            <img src="icons/heart.svg" alt="Like"></img>
+          </div>
+          <div class="interact play">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
+              <path  d="M50 6.00001c-24.2767 0-44.0000001 19.7233-44.0000001 44.00001 0 24.2767 19.7233001 44 44.0000001 44s44-19.7233 44-44C94 25.72331 74.2767 6.00001 50 6.00001zm0 4c22.1149 0 40 17.8851 40 40.00001 0 22.1149-17.8851 40-40 40s-40.0000001-17.8851-40.0000001-40C9.9999999 27.88511 27.8851 10.00001 50 10.00001zm-15 21v38.00001l34-19-34-19.00001z" overflow="visible" />
+            </svg>
```
Comment (ericaglimsholt) [inline code/src/Components/Album.js]: Why isn't this svg in the icons folder as the others?

Comment (tehvicke) [reply code/src/Components/Album.js]: I was experimenting a bit with svg styling hence why I ended up lik this :)

**Comment by tehvicke** [comment]
Hej Erica - nu har jag lagt till de kraven som saknas, tusen tack!

File: code/src/Components/Album.js (hunk 1 above)
Comment (tehvicke) [reply code/src/Components/Album.js]: good point, updated!

**Pull request closed without merging**